client/tailscale/tailscale.go

```diff
@@ -179,10 +183,6 @@ func send(ctx context.Context, method, path string, wantStatus int, body io.Read
 		return nil, err
 	}
 	if res.StatusCode != wantStatus {
-		if res.StatusCode == 403 {
```

Comment:
any harm in leaving this? Are there any code paths that use send directly instead of `doLocalRequestNiceError` and would no longer get AccessDeniedErrors?

Reply:
`tailscale file get .` calls doLocalRequestNiceError() instead of going through send() because it wants access to `res.ContentLength` as far as I can tell, so that's what prompted the move of this bit down the stack.